### PR TITLE
[Fix] indentation errors leading to codeblocks in verbatim in PDF

### PR DESCRIPTION
**Pull request by jpfeuffer**
... tutorial. Also added some missing sections.

### CODE REVIEW

**Comment by jpfeuffer** [comment]
I have not tested what this does to the html documentation, yet.
And to avoid such things we should define clear rules on how to write the documentation. How to indent things, which markup/down is allowed, how to put references, when to use section etc. etc. There were a lot of sections missing. Pages are not enough for PDF but only work for the html. And still I did not get all of the references to work yet.

TLDR there is much more to it than just this quick fix but I do not have the time now. I think nobody really had the PDF in mind when creating the doxygen files.
